test/contents/autofillTest.js

```diff
@@ -213,7 +213,7 @@ describe('Autofill', function () {
         .waitForTextValue('.creditCardPExpirationDate',
           (expMonth < 10 ? '0' + expMonth.toString() : expMonth.toString()) + '/' + expYear.toString())
     })
-    it('autofills the credit card', function * () {
+    it.skip('autofills the credit card', function * () {
```

Comment:
Update of https://github.com/brave/browser-laptop/issues/5389 is necessary I think.